examples/cactus-example-supply-chain-frontend/src/app/bamboo-harvest/bamboo-harvest-detail/bamboo-harvest-detail.page.ts

```diff
@@ -10,6 +10,18 @@ import { BambooHarvest } from "@hyperledger/cactus-example-supply-chain-business
 import { QUORUM_DEMO_LEDGER_ID } from "../../../constants";
 import { Logger, LoggerProvider } from "@hyperledger/cactus-common";
 
+import { RuntimeError } from "run-time-error";
+
+function isBambooHarvest(value: unknown): value is BambooHarvest {
+  const harvest = value as BambooHarvest;
+  return (
+    typeof harvest.id === "string" &&
+    typeof harvest.location === "string" &&
+    typeof harvest.startedAt === "string" &&
+    typeof harvest.endedAt === "string" &&
+    typeof harvest.harvester === "string"
+  );
+}
```

Comment:
@adrianbatuto This is not type safe at runtime because you are doing an unchecked cast again. Please take a look at how other user-defined type guards are usually written in the code, such as packages/cactus-core-api/src/main/typescript/plugin/keychain/is-i-plugin-keychain.ts